packages/flutter/lib/src/widgets/sliver_persistent_header.dart

```diff
@@ -224,7 +231,13 @@ class _SliverFloatingPersistentHeader extends _SliverPersistentHeaderRenderObjec
 
   @override
   _RenderSliverPersistentHeaderForWidgetsMixin createRenderObject(BuildContext context) {
-    return new _RenderSliverFloatingPersistentHeaderForWidgets();
+    return new _RenderSliverFloatingPersistentHeaderForWidgets()
```

Comment:
Weaving the constructor parameter through all of the intermediate classes (as a positional parameter because of https://github.com/dart-lang/sdk/issues/15101) seemed like it would only make things more difficult to understand.